.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file shouldn't be in the repo, it may interfere with other developers.  The .vscode path should really be in .gitignore, but you don't have to do that.